src/cover-hero-media.tsx

```diff
@@ -0,0 +1,35 @@
+'use client';
```

Comment:
extracted this part to a different file because it's te interactive part of the component, hence we need `"use client"`. This way the rest of the component is a RSC and will not contribute to bundle size in the browser.